api/v1alpha1/condition_types.go

```diff
@@ -25,6 +25,14 @@ const (
 	// the progressive sync stages have been completed
 	StagesCompleteReason string = "StagesCompleted"
 
+	// StagesInitializedReason represent the fact that all
+	// the progressive sync stages have been initialized
+	StagesInitializedReason string = "StagesInitialized"
```

Comment:
We don't appear to need this any longer?

Reply:
Good catch, thanks

Reply:
Removed